docs/decisions/0002-lti-1p3-variables.rst

```diff
@@ -0,0 +1,123 @@
+LTI 1.3 variable definition
+---------------------------
+
+Status
+======
+
+Provisional
+
+Context
+=======
+
+The LTI 1.3 specification defines multiple variables related to course, context, problem, student id and PII.
+
+The goal of this ADR is to provide well defined mappings from variables available in the platform and
+variables required by LTI 1.3 launches.
+
+Decisions
+=========
+
+The following section go over required and optional variables in each supported claim in the LTI 1.3 implementation.
+This ADR should be updated when new claims are added.
+
+Deployment ID
+~~~~~~~~~~~~~
++---------------------------------------------------------+------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
BTW, .rst has a table style called list-table: https://docutils.sourceforge.io/docs/ref/rst/directives.html#list-table

It lets you author the file as a list, but be rendered as a table.

Reply:
@nedbat Much better, thanks for the suggestion!